composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "partnermarketing/rebrandly-sdk",
+    "name": "siluet/rebrandly-sdk",
```

Comment:
You're welcome to start your own fork of the project but we can't merge this rename into our version. We are, of course, happy to merge your `->export()` fix in though if you remove this change :smile: